mbof/views.py

```diff
@@ -28,9 +29,24 @@ def log_action(request):
     # EXAMPLE - dovek : GET /api/events/current/
     message = '%s : %s %s' % (me, method, url)
     logger.info(message)
-    
 
-class UserViewSet(viewsets.ModelViewSet):
+
+class OcellusViewSet(mixins.CreateModelMixin,
+                       mixins.RetrieveModelMixin,
+                       mixins.UpdateModelMixin,
+                       # mixins.DestroyModelMixin,
```

Comment:
Remove this commented out class reference.  Add some text that explains why this class was created.  (That you wanted to leave out the delete capabilities provided by `mixins.DestroyModelMixin`.)